deephub/models/feeders/tfrecords/meta.py

```diff
@@ -54,9 +54,11 @@ def generate_for(cls, fpath: Path) -> 'TFRecordFileInfo':
 
         """
         file_size = fpath.stat().st_size
```

Comment:
The compression type here, must also be an argument to this function. It must be passed as a command line argument from the cli. Add a new argument, compression_type, with default value equal to empty.